src/thunderbots/firmware/main/dribbler.c

```diff
@@ -37,9 +39,9 @@
 #define THERMAL_WARNING_STOP_TEMPERATURE_WINDING (THERMAL_WARNING_START_TEMPERATURE_WINDING - 15.0f) // °C—chead
 #define THERMAL_WARNING_STOP_ENERGY_WINDING ((THERMAL_WARNING_STOP_TEMPERATURE_WINDING - THERMAL_AMBIENT) * THERMAL_CAPACITANCE_WINDING) // joules
 
-#define DRIBBLER_SPEED_BUFFER_ZONE 200 // RPM
+#define DRIBBLER_SPEED_BUFFER_ZONE 250 // RPM
 
-#define MAX_DRIBBLER_CURRENT 5 // Limit the max current the dribbler can draw when stalled
+#define MAX_DRIBBLER_CURRENT 4.0 // Limit the max current the dribbler can draw when stalled
```

Comment:
If we are throwing 'f's on the end of all the previous defines, let's also do it here.

Reply:
fixed